sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoByteUtils.java

```diff
@@ -96,6 +164,23 @@ public Row apply(byte[] input) {
     };
   }
 
+  public static SerializableFunction<Row, byte[]> getRowToProtoFromSchemaBytes(
```

Comment:
nit: 
```suggestion
  public static SerializableFunction<Row, byte[]> getRowToProtoBytesFromSchema(
```
seems to follow the other names closer.